.github/workflows/build.yaml

```diff
@@ -70,3 +77,6 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/arm64/v8
           push: true
           tags: quay.io/abhishekvrshny/propeller:latest
+      - name: Set Build Timestamp
+        id: date
+        run: echo "date=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
```

Comment:
how does this step help?

Reply:
Again used for logging information, can consider removing since metadata is also removed.

Reply:
fixed that as well since metadata is not required, logging timestamp serves no purpose